Javascript/homework/responses/2.2-body.md

```diff
@@ -4,4 +4,9 @@
 
 #### Task 2: Styling the body
 
-In the body class, center all items, select a font and size for your text, and change the background color of the page.
\ No newline at end of file
+In your CSS file, define a body class and set the display to flex.  This will make sure all elements fit the screen appropriately. Next, use the `justify-content`  and `align-items` attributes to center your items on the y axis and the x axis respectively. You may also want to set the `min-height` to `100vh` so that height of your content will always be relative to the size of the screen.
```

Comment:
How do you define a body class? Should this be googled as well to display to flex?